README.md

```diff
@@ -1,283 +1,487 @@
+  </a>
+  <br>
+  <br>
+	<a href="https://npmjs.com/package/webpack">
+		<img src="https://img.shields.io/npm/dm/webpack.svg">
+	</a>
+	<a href="https://opencollective.com/webpack#backer">
+		<img src="https://opencollective.com/webpack/backers/badge.svg">
+	</a>
+	<a href="https://opencollective.com/webpack#sponsors">
+		<img src="https://opencollective.com/webpack/sponsors/badge.svg">
+	</a>
+	<a href="https://gitter.im/webpack/webpack">
+		<img src="https://badges.gitter.im/webpack/webpack.svg">
+	</a>
+  <h1>webpack</h1>
+  <p>
+    webpack is a bundler for modules. The main purpose is to bundle JavaScript files for usage in a browser, yet it is also capable of transforming, bundling, or packaging just about any resource or asset.
+  <p>
+</div>
+
+<h2 align="center">Install</h2>
+
+```bash
+npm install --save-dev webpack
+```
 
-[documentation](https://webpack.github.io/docs/?utm_source=github&utm_medium=readme&utm_campaign=top)
+<h2 align="center">Introduction</h2>
 
-# Introduction
+> The README reflects webpack v2.x, webpack v1.x [documentation can be found here](https://webpack.github.io/docs/?utm_source=github&utm_medium=readme&utm_campaign=top).
 
 webpack is a bundler for modules. The main purpose is to bundle JavaScript
 files for usage in a browser, yet it is also capable of transforming, bundling,
 or packaging just about any resource or asset.
 
+**TL;DR**
 
-**TL; DR**
-
-* Bundles both [CommonJS](http://wiki.commonjs.org/) and [AMD](https://github.com/amdjs/amdjs-api/wiki/AMD) modules (even combined).
+* Bundles [ES Modules](http://www.2ality.com/2014/09/es6-modules-final.html), [CommonJS](http://wiki.commonjs.org/) and [AMD](https://github.com/amdjs/amdjs-api/wiki/AMD) modules (even combined).
 * Can create a single bundle or multiple chunks that are asynchronously loaded at runtime (to reduce initial loading time).
-* Dependencies are resolved during compilation reducing the runtime size.
-* Loaders can preprocess files while compiling, e.g. coffeescript to JavaScript, handlebars strings to compiled functions, images to Base64, etc.
+* Dependencies are resolved during compilation, reducing the runtime size.
+* Loaders can preprocess files while compiling, e.g. CoffeeScript to JavaScript, Handlebars strings to compiled functions, images to Base64, etc.
 * Highly modular plugin system to do whatever else your application requires.
 
-# Getting Started
-
-Check out webpack's [documentation](https://webpack.github.io/docs/?utm_source=github&utm_medium=readme&utm_campaign=trdr) for quick Getting Started guide, in-depth usage,
-tutorials and resources.
-
-# Installation
-
-project:
-`npm install webpack --save-dev`
-
-global:
-`npm install webpack -g`
+### [Get Started](https://webpack.js.org/get-started/)
 
-Usage:
-https://webpack.github.io/docs/tutorials/getting-started/
+Check out webpack's [docs](https://webpack.js.org/guides/) for quick Getting Started guide, in-depth usage, tutorials and resources.
```

Comment:
hmm.. I'm not really happy with this section in general 😛. It should reflect the docs top-level structure somehow/better. I willl refactor that section when I something better in mind, but the various plugins && loaders docs first. Otherwise 👍 on the link change